packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -306,15 +310,20 @@ class MethodChannel {
   @optionalTypeArgs
   Future<T> invokeMethod<T>(String method, [ dynamic arguments ]) async {
     assert(method != null);
-    final ByteData result = await binaryMessenger.send(
-      name,
-      codec.encodeMethodCall(MethodCall(method, arguments)),
-    );
-    if (result == null) {
-      throw MissingPluginException('No implementation found for method $method on channel $name');
+    try {
+      _pendingChannelInvokeMethodCount += 1;
```

Comment:
This should go immediately before the `try`

Reply:
Done.